.github/workflows/test.yml

```diff
@@ -8,16 +8,15 @@ on:
 
 env:
   PYTHONUNBUFFERED: 1
-  PYTEST_ADDOPTS: "--color=yes"
 
 jobs:
   conda-install:
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
-        python-version: ['3.7', '3.8', '3.9', '3.10']
+        os: ["ubuntu-latest", "macos-latest"]
+        python-version: ['3.9']
```

Comment:
revert this change; I only made this change to make the builds faster during development